osbs/build/user_params.py

```diff
@@ -80,10 +85,17 @@ def set_params(self,
                    flatpak=None, reactor_config_map=None, reactor_config_override=None,
                    yum_repourls=None, signing_intent=None, compose_ids=None,
                    isolated=None, scratch=None, parent_images_digests=None,
+                   tags_from_yaml=None, additional_tags=None, container_yaml=None,
+                   git_commit_depth=-1,
                    **kwargs):
         self.git_uri.value = git_uri
         self.git_ref.value = git_ref
         self.git_branch.value = git_branch
+        self.git_commit_depth.value = git_commit_depth
+        self.tags_from_yaml.value = tags_from_yaml
+        self.additional_tags.value = additional_tags or {}
+        self.container_yaml.value = container_yaml or {}
```

Comment:
I haven't seen this attribute used in this PR, only in init methods. What it does and what is type of that attr?

Reply:
container_yaml is the existing RepoInfo's RepoConfiguration's container member, which is the yaml.load of REPO_CONTAINER_CONFIG.  So it should be the entire existing container.yaml, read once and passed around to atomic reactor so atomic reactor doesn't need to reclone the repo just to read the container.yaml.

I am currently running into the issue of how to pass something from user_params back up to input_osv3.